src/components/Definitions/Definitions.scss

```diff
@@ -20,10 +20,25 @@ limitations under the License.
 }
 
 main {
-  .bx--structured-list {
-    background-color: white;
+  .bx--structured-list-th {
+    background-color:  #7190c1;
```

Comment:
Can we leave out the blue background and the cell borders? What we discussed on the call was just applying the zebra striping and titles. Also where we do specify colours we should use the Carbon design tokens (variables) where possible instead of hard coding them. See https://www.carbondesignsystem.com/guidelines/color/usage for details of the available colour tokens.